custom_components/tuya_local/devices/eurom_sani_bathroom_towel_radiator.yaml

```diff
@@ -0,0 +1,112 @@
+name: Eurom Sani-Bathroom-Radiator Towel Wifi heater
+primary_entity:
+  entity: climate
+  translation_key: heater
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: "r"
+              value: "heat"
+            - dps_val: "t"
+              value: "dry"
+              icon: "mdi:hanger"
+    - id: 2
+      type: integer
+      name: temperature
+      range:
+        min: 0
+        max: 37
+      mapping:
+        - constraint: mode
+          conditions:
+            - dps_val: "t"
+              value_redirect: towel_dry_temperature
+              range:
+                min: 40
+                max: 70
+    - id: 333
+      # dummy variable for current temperature of
```

Comment:
That's the point, I need to create an entity that doesn't exist, since the others (heater_temperature:105 and room_temperature:3) are already used. And the "current_temperature" that is used depends on the operation mode:2. I could use one of the above as the base entity (let's say heater_temperature), but then that one behaves rather strange in the logging. So the current_temperature entity is the actual temperature as used by the internal control loop, corresponding to the mode of operation. 
If there is another option to create an entity without a dps id, I'd like to use that, but for now, this seems to be the only workaround.

Reply:
Or is it possible to use the same dp id in both the primary and secondary entities? That would allow the actual values to be used (heater surface temperature and room temperature) as individual sensors (nice for making graphs) and use either of the two as the current temperature in the climate/heater device. That would be most elegant and correct I think.

Reply:
Yes, it is possible to reuse the same dp in another entity, or even in the same entity with a different name.  Making secondary sensor entities for both temperatures would make them easier to work with than as non-standard attributes on the climate entity.

Reply:
I made a change accordingly.